airbyte-cdk/python/unit_tests/sources/file_based/scenarios/csv_scenarios.py

```diff
@@ -1450,7 +1487,6 @@
         }
     )
     .set_expected_discover_error(AirbyteTracedException, FileBasedSourceError.SCHEMA_INFERENCE_ERROR.value)
-    .set_expected_records([])
```

Comment:
These 2 scenarios were scenarios that fail discover. They previously had "0 expected records", but fail silently if sent through the read tests. Instead, they are still discover scenarios, but no longer read scenarios - if discover fails, we don't expect the read to succeed. 

These in particular don't assert any specific expected errors, which hints toward silent failure. If we got these cases to yell more aggressively about their errors, we could bring them back as read tests with expected errors.